.ci/pytorch/test.sh

```diff
@@ -1225,6 +1225,17 @@ test_linux_aarch64(){
        inductor/test_torchinductor_codegen_dynamic_shapes inductor/test_torchinductor_dynamic_shapes --verbose
 }
 
+test_operator_benchmark() {
+  TEST_REPORTS_DIR=$(pwd)/test/test-reports
+  TEST_DIR=$(pwd)
+  end_core=$(( CORES-1 ))
+  cd benchmarks/operator_benchmark/pt_extension
+  python setup.py install
+  cd "${TEST_DIR}"
+  taskset -c 0-"$end_core" python -m benchmarks/operator_benchmark/benchmark_all_test.py --output "${TEST_REPORTS_DIR}/operator_benchmark.csv"
```

Comment:
Do we need to test all test cases? all just pick some subsets? And let's think about how to check the performance regression in this function